content/1. Leertaken/WEBDEV/Security/Workshops/10. Workshop Git/3. Git Forks & Branches.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 3. Git Forks & Branches
+taxonomie: None
+tags: 
+draft: true
+---
+## Opdracht 1: Forken
+In de volgende opdrachten gaan we dieper in op de functies die Git(Hub) biedt. De eerste stap is het forken van een repository.
+
+> [!info] Een fork is een kopie van een repository. Door een repository te forken, kun je vrij experimenteren met wijzigingen zonder dat je het originele project beïnvloedt.
```

Comment:
Dit kan je weglaten, dit komt in OI aan bod. Dit geldt ook voor die andere info blokken die echt git uitleggen (zoals prs, forks en branches)